sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -105,6 +105,18 @@ class BlobClient(StorageAccountHostsMixin):  # pylint: disable=too-many-public-m
         account URL already has a SAS token. The value can be a SAS token string, and account
         shared access key, or an instance of a TokenCredentials class from azure.identity.
         If the URL already has a SAS token, specifying an explicit credential will take priority.
+    :keyword int max_block_size: The maximum upload size used for a block blob. If the
+        block blob exceeds this size, it will be uploaded in chunks. Defaults to 4*1024*1024.
+    :keyword int max_single_put_size: If the blob size is less than max_single_put_size, then the blob will be
+        uploaded with only one http PUT request. If the blob size is larger than max_single_put_size,
+        the blob will be uploaded in chunks. Defaults to 64*1024*1024.
+    :keyword int min_large_block_upload_threshold: The minimum upload size
+        threshold used for a block blob. Defaults to 4*1024*1024+1.
```

Comment:
min_large_block_upload_threshold: the minimum chuck size to use the memory efficient algorithm when upload a block blob